source/source.js

```diff
@@ -42,6 +46,24 @@ var installPromiseMatchers;
 
     $scope.$apply(); // Trigger Promise resolution
 
+    // Trigger $httpBackend flush if any requests are pending
+    try {
+      $httpBackend.flush();
+    } catch (err) {
+      if (err.message !== 'No pending request to flush !') {
```

Comment:
String-matching on an error message seems very questionable. (It would be different to match an error code, since those are implied to be relatively stable, but this... I feel uneasy about.)

Granted neither of these strings have changed since _Oct 22, 2013_ so maybe this is just me being overly cautious...

I don't use this library anymore since I haven't worked with Angular 1.x for over a year now. So I guess I'll defer to your judgement in terms of the convenience trade-off here vs the "risk" of the inline string. If any unexpected problems are reported against this behavior though, I'll have to remove it. Seem reasonable?